docs/resources/recipient.md

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_recipient Resource
+
+Within a metastore, Unity Catalog provides the ability to create a recipient to attach delta shares to.
+
+A `databricks_recipient` is contained within [databricks_metastore](metastore.md) and can contain a list of shares.
+
+## Example Usage
+
+```hcl
+resource "databricks_recipient" "db2open" {
+  name = "db2open-recipient"
+  comment = "made by terraform"
+  authentication_type = "TOKEN"
+  sharing_code = "my code "
+  ip_access_list {
+    allowed_ip_addresses = ["0.0.0.0/0"]
+  }
+}
+
+resource "databricks_recipient" "db2db" {
+  name = "sri-terraform-test-db2db-recipient"
+  comment = "made by terraform"
+  authentication_type = "DATABRICKS"
+  data_recipient_global_metastore_id = "<cloud>:<region>:<guid>"
```

Comment:
can you expand the example with `databricks_metastore` resource to refer the ID from it?